04 Event Emitters/assets/js/app/components/TableComponent.js

```diff
@@ -0,0 +1,59 @@
+(function ComponentInitializator($, React, App) {
+  'use strict';
+
+  var createCell = function (type, text) {
+    return React.createElement(type, null, text);
+  };
+
+  var createRow = function (cell, index, contact) {
+    return React.createElement('tr', { key: index },
+      createCell(cell, contact.name),
+      createCell(cell, contact.phone),
+      createCell(cell, contact.email)
+    );
+  };
+
+  var createThead = function () {
+    var contactTemplate = {
```

Comment:
The previous functions you use createRow and createCell, and here you start to use createThead. I think that is better to be homogenous naming functions, so I will use instead createHead. I think it is explicit enough on this case.